sdk/file_sdk/XMPFileFormat/xmp_stream.h

```diff
@@ -0,0 +1,106 @@
+#ifndef XMP_STREAM_H
+#define XMP_STREAM_H
+
+#include <IStream.h>
+#include <XMP_IO.hpp>
+#include <XMP_Const.h>
+#include <CryptoAPI.h>
+#include <sstream>
+
+using rmscrypto::api::IStream;
+using std::static_pointer_cast;
+using std::make_shared;
+
+namespace mip {
+namespace file {
+
+class XMPStream : public XMP_IO
```

Comment:
XMPIOOverIStream - what do you think?

Reply:
I think it looks weird but the meaning is better

---
In reply to: [136420037](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136420037) [](ancestors = 136420037)